src/countries/dto/update-country.dto.ts

```diff
@@ -0,0 +1,4 @@
+export class UpdateCountryDto {
+  name: string;
+  logo: string;
+}
```

Comment:
```suggestion
}

```

This is a temporary fix only